apps/gauzy/src/app/pages/jobs/proposal-template/proposal-template/proposal-template.component.html

```diff
@@ -127,12 +127,21 @@ <h4>
 	<ng-template ngxPermissionsOnly="ORG_PROPOSAL_TEMPLATES_VIEW">
 		<div class="table-scroll-container">
 			<ng2-smart-table
+				*ngIf="!noData"
 				style="cursor: pointer"
 				[settings]="smartTableSettings"
```

Comment:
@lawphilenz you should use ternary condition
```suggestion
				[settings]="noData ? smartTableSettingsNoData : smartTableSettings"
```

Reply:
@adkif already tried using ternary but it didn't worked. it has some race condition. let me make some more changes. thanks.